### PR TITLE
ViewStudentList

### PR DESCRIPTION
**Pull request by tuannheddy97**

### CODE REVIEW

**Comment by tuannheddy97** [comment]
image

File: views/view-student-list.pug [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    .card
+        .card-body
+            form.form-inline.my-2.my-lg-0
+                input.form-control-sm.form-control.mr-sm-2(type='search' placeholder='Search' aria-label='Search')
+                button.btn.btn-sm.btn-primary.my-2.my-sm-0(type='submit') Search
+
+    ul.list-group.list-group-horizontal-md
+        li.list-group-item(class='item-action active') Class 1
+        li.list-group-item(class='item-action ') Class 2
+        li.list-group-item(class='item-action ') Class 3
```
Comment (trantanhdev) [inline views/view-student-list.pug]: Remove these code we don't manage with class

File: views/view-student-list.pug [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+    ul.list-group
```
Comment (trantanhdev) [inline views/view-student-list.pug]: add class ".list-group-flush"

File: views/view-student-list.pug [hunk 3]
```diff
@@ -0,0 +1,46 @@
+extends role_layout
```
Comment (trantanhdev) [inline views/view-student-list.pug]: name this file "student_list"

File: views/view-student-list.pug [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+block page_content
+
+    .card
+        .card-body
+            form.form-inline.my-2.my-lg-0
+                input.form-control-sm.form-control.mr-sm-2(type='search' placeholder='Search' aria-label='Search')
+                button.btn.btn-sm.btn-primary.my-2.my-sm-0(type='submit') Search
```
Comment (trantanhdev) [inline views/view-student-list.pug]: Remove these code. We don't have this feature.

File: views/view-student-list.pug [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    ul.list-group.list-group-horizontal-md
+        li.list-group-item(class='item-action active') Class 1
+        li.list-group-item(class='item-action ') Class 2
+        li.list-group-item(class='item-action ') Class 3
+
+
+    ul.list-group
+        li.list-group-item Student 1
+            a(href='')
+                img.img-thumbnail(src='../imgs/pdf.jpg' alt='pdf' width="200" class='rounded float-right')
```
Comment (trantanhdev) [inline views/view-student-list.pug]: Remove all images. We will take care of it later.

File: views/view-student-list.pug (hunk 2 above)
Comment (trantanhdev) [inline views/view-student-list.pug]: Move whole this ul element into card-body

**Comment by trantanhdev** [comment]
Screen Shot 2020-03-21 at 10 37 08
UI just simple like this.

**Pull request closed without merging**